### ISSUE TITLE
create knife.rb if not specified

### ISSUE THREAD

**Issue by erikh** [fnichol/knife-server#25]
feature request - if -c isn't specified, mktemp, dump a templated knife.rb with the host in there and pems that live in the same dir, barf the path to stdout when knife server completes.

I have code for this in chef-workflow already and having this here helps new users get up to speed faster as well. the template is pretty integrated but communicates the idea:

https://github.com/chef-workflow/chef-workflow/blob/master/lib/chef-workflow/support/knife.rb#L31

basically KISS on the standard config and let people migrate from there.

it'd also be really really nice for testing on ec2 where building a new machine almost always requires specifying the new hostname in two spots.

**Comment by fnichol** [comment]
Yep, this is a good idea. I didn't want it to hold up a 1.0 release any more but should be able to get to this next.

**Comment by erikh** [comment]
yeah, no problem. I'll see if i can bang something out over the next couple
of weeks if you don't get to it first.

**Comment by fnichol** [comment]
Heya @erikh I don't think is is really blocking anybody and if you (or anyone else) wants to take a crack at it, don't hesitate to open a PR. For the meantime I'm going to tend this garden and close the issue out.

Thanks for continuing to be an awesome contributor and collaborator :smile:
